src/Http/EmptyStream.php

```diff
@@ -0,0 +1,55 @@
+<?php
```

Comment:
no, it's necessary. ResponseGenerator middleware uses it because Diactorososos has a null body if you didn't explicitly call withBody() on a response, which can cause null pointer exceptions places that expect getBody() to return a body..

Reply:
added doc comments